README.md

```diff
@@ -539,15 +539,17 @@ Create a new Adobe I/O App
 ```
 USAGE
   $ aio app init [PATH] [-v] [--version] [--install] [-y] [--login] [-e <value>... | -t <value>... | --repo
-    <value>] [--standalone-app |  | ] [-w <value> | -i <value>] [--confirm-new-workspace] [--use-jwt] [--github-pat
-    <value> ] [--linter none|basic|adobe-recommended]
+    <value>] [--standalone-app |  | ] [--template-options <value>] [-o <value> | -i <value> | ] [-p <value> |  | ] [-w
+    <value> |  | ] [--confirm-new-workspace] [--use-jwt] [--github-pat <value> ] [--linter none|basic|adobe-recommended]
 
 ARGUMENTS
   PATH  [default: .] Path to the app directory
 
 FLAGS
   -e, --extension=<value>...        Extension point(s) to implement
   -i, --import=<value>              Import an Adobe I/O Developer Console configuration file
+  -o, --org=<value>                 Specify the Adobe Developer Console Org to init from (orgId, or orgCode)
+  -p, --project=<value>             Specify the Adobe Developer Console Project to init from (projectId, or projectName)
```

Comment:
Will both `projectId` or `projectName` work in the same way?
Same question for `orgId` and `orgCode`.

Reply:
Yes, it does a search on both id or name. This is already existing functionality, just wasn't documented.

1. https://github.com/adobe/aio-cli-lib-console/blob/d4fae778cd091ffef66e8e54669a19fb39602c20/lib/index.js#L80
2. https://github.com/adobe/aio-cli-lib-console/blob/d4fae778cd091ffef66e8e54669a19fb39602c20/lib/index.js#L132

Where the selection happens:
1. https://github.com/adobe/aio-cli-lib-console/blob/d4fae778cd091ffef66e8e54669a19fb39602c20/lib/pure-helpers.js#L36-L41
2. https://github.com/adobe/aio-cli-lib-console/blob/d4fae778cd091ffef66e8e54669a19fb39602c20/lib/pure-helpers.js#L54-L59